sdk/eventhub/Azure.Messaging.EventHubs/stress/README.md

```diff
@@ -0,0 +1,39 @@
+# Event Hub Stress Tests
+This is a preliminary README. It demonstrates how to use the stress tests in the current state.
+
+In order to run the stress tests locally, the necessary resource connections need to be input through the command line interface. Test runs can call any of the following tests:
+- EventProducerTest : "EventProd"
+- EventBufferedProducerTest : "EventBuffProd"
+- BurstBufferedProducerTest : "BurstBuffProd"
+- ConcurrentBufferedProducerTest : "ConcurBuffProd"
+
+## Local Stress Test Runs
+### Setting up resources and packages
+```cmd
+(env) ~/stress/src> dotnet clean
+(env) ~/stress/src> dotnet build
+```
+
+### Running Tests
+When tests are run locally, Azure resources need to be created prior to running the test. This can be done through the Azure CLI, an ARM template or bicep file, or the Azure Portal. The user is required to input the connection strings upon request on the command line when the test is being run. For more information about what resources are needed for each test, see the "Scenario Information" section below. 
```

Comment:
@ckairen I think we should also support a "deploy resources only" type mode to make local testing easier with resources.